### PR TITLE
Added bug_report Template according to your instruction

### PR DESCRIPTION
**Pull request by parthiv11**
## Change description

Added bug report template in yaml

### CODE REVIEW

**Comment by parthiv11** [comment]
@evantahler  Sorry for being so late but now you can merge this PR

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    label: Is there an existing issue for this?
+    description: Please search to see if an issue already exists for the bug you encountered.
+    options:
+    - label: I have searched the existing issues
+      required: true
+      
+- type: textarea
+  attributes:
+    label: Current Behavior
+    description: A concise description of what you're experiencing.
```
Comment (evantahler) [inline .github/ISSUE_TEMPLATE/bug_report.yml]:
```suggestion
    description: A concise description of what you're experiencing.  Please include specific steps or a sample project to reproduce the error you are seeing.
```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 2]
```diff
@@ -0,0 +1,69 @@
+name: 🐞 Bug report
+description: Create a bug report to help us improve Grouparoo
+title: "[Bug]: "
+labels: bug
+
+body:
+- type: markdown
+  attributes:
+    value: |
+      Thanks for taking the time to fill out this bug report! Please fill the form in English!
```
Comment (evantahler) [inline .github/ISSUE_TEMPLATE/bug_report.yml:10]: This intro can probably be removed.

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    placeholder: |
+      If applicable, add the full stack trace you see along with the error.
+    render: markdown
+  validations:
+    required: false
+
+
+- type: textarea
+  attributes:
+    label: Screenshots or Environment ,OS , Grouparoo Version etc
```
Comment (evantahler) [inline .github/ISSUE_TEMPLATE/bug_report.yml]:
```suggestion
    label: Environment
```

Comment (evantahler) [reply .github/ISSUE_TEMPLATE/bug_report.yml]: There were some mistakes with the commas, etc

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+       - Browser [e.g. chrome, safari]
+       - Grouparoo Version [e.g. 1.2.3]
+       - Node.js Version [e.g. 1.2.3]
+
+  validations:
+    required: true
+    
+- type: textarea
+  attributes:
+    label: Additional  context
```
Comment (evantahler) [inline .github/ISSUE_TEMPLATE/bug_report.yml]:
```suggestion
    label: Additional context
```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+    description: |
+      Links? References? Anything that will give us more context about the issue you are encountering!
```
Comment (evantahler) [inline .github/ISSUE_TEMPLATE/bug_report.yml]:
```suggestion
    description: |
      Links? Screenshots? References? Anything that will give us more context about the issue you are encountering!
```

**Review by evantahler — CHANGES_REQUESTED** [review]
Getting closer! Thank you for the update!

**Comment by parthiv11** [comment]
@evantahler  Now It is OK Merge this PR

**Review by evantahler — APPROVED** [review]

**Pull request merged**